packages/mask/src/plugins/NextID/base.ts

```diff
@@ -9,7 +9,7 @@ export const base: Plugin.Shared.Definition = {
     publisher: { name: { fallback: 'Mask Network' }, link: 'https://mask.io/' },
     enableRequirement: {
         architecture: { app: false, web: true },
-        networks: { type: 'opt-out', networks: {} },
+        networks: { type: 'opt-out', networks: { [CurrentSNSNetwork.Instagram]: true } },
```

Comment:
does next id work on Facebook and minds?

Reply:
Yes, it should work on them.